mantidimaging/core/io/utility.py

```diff
@@ -153,11 +152,73 @@ def _alphanum_key_split(path_str):
     return [int(c) if c.isdigit() else c for c in alpha_num_split_re.split(path_str)]
 
 
-if __name__ == '__main__':
-    import doctest
+def get_prefix(path: str, separator="_"):
+    return path[:path.rfind(separator)]
```

Comment:
I prefer `path.rpartition(separator)[0]`

Reply:
This is just a refactored method from somewhere else.